shared/components/TicketList/index.js

```diff
@@ -1,26 +1,44 @@
 import React, { PropTypes } from 'react';
-import { formatDate } from '../../utilities/date';
+import moment from 'moment';
+import { formatDate, isAfter } from '../../utilities/date';
 import { ticketType } from '../../prop-types/speaker-type';
 
+export function isSoldOut(ticket) {
+  const currentDateTime = moment();
+  if (!ticket.available && isAfter(currentDateTime, ticket.releaseDate)) {
+    return true;
+  }
+  return false;
+}
+
 const TicketPrice = ({ ticket }) => {
+  const isTicketSoldOut = isSoldOut(ticket);
   return (
-    <strong>{ticket.available ? `£${ticket.price}` : 'SOLD OUT'}</strong>
+    <strong>{!isTicketSoldOut ? `£${ticket.price}` : 'SOLD OUT'}</strong>
```

Comment:
isTicketSoldOut should be isTicketAvailable - because it's not just about if they are sold or not